test/test_jit_fuser.py

```diff
@@ -796,7 +796,10 @@ def create(self, x):
         x = torch.zeros([3, 4, 5], dtype=torch.float, device='cuda')
         m = M()
         out1 = m.create(x)
+        start_execs = torch._C._jit_get_trigger_value("cuda_codegen_executed")
```

Comment:
Maybe add a TODO to refactor CudaCodeGenExecuted and use it here?

Reply:
Sure, let me go ahead and do that in a follow up.